cmd/otelcol/main.go

```diff
@@ -67,7 +68,7 @@ func main() {
 
 	// Allow dumping configuration locally by default
 	// Used by support bundle script
-	os.Setenv(configServerEnabledEnvVar, "true")
+	_ = os.Setenv(configServerEnabledEnvVar, "true")
```

Comment:
Since we're now explicitly ignoring all these errors I wonder if it's better to `log.Fatal`?

Reply:
(Just trying to get rid of Goland warnings while I'm in here) It looks like `Setenv` only returns an error if the key is either empty, contains an equals character, or a has a null byte, which I don't think we're in danger of since it's hard coded. But happy to do so if you'd prefer.

Reply:
Appears the remaining error condition in the linux syscall is oom, so logging would be impossible anyway.  no need to change here I guess.